src/nemos/solvers/_solver_registry.py

```diff
@@ -0,0 +1,36 @@
+"""Registry for mapping from solver name to concrete implementation."""
+
+from typing import Type
+
+from ._jaxopt_solvers import (
+    JaxoptBFGS,
+    JaxoptGradientDescent,
+    JaxoptLBFGS,
+    JaxoptNonlinearCG,
+    JaxoptProximalGradient,
+)
+
+# from ._optimistix_solvers import OptimistixBFGS, OptimistixNonlinearCG
+from ._svrg import WrappedProxSVRG, WrappedSVRG
+
+# from ._optax_optimistix_solvers import (
```

Comment:
what's going on here (and other comments in this file)?

Reply:
- [ ] Remove them